src/app/modules/main/chat_section/module.nim

```diff
@@ -1013,9 +1013,10 @@ method onNewMessagesReceived*(self: Module, sectionIdMsgBelongsTo: string, chatI
   let messageBelongsToActiveChat = self.controller.getActiveChatId() == chatIdMsgBelongsTo
 
   singletonInstance.globalEvents.showMessageNotification(notificationTitle, plainText, sectionIdMsgBelongsTo,
-    self.controller.isCommunity(), messageBelongsToActiveSection, chatIdMsgBelongsTo, messageBelongsToActiveChat,
-    message.id, notificationType.int, chatTypeMsgBelongsTo == ChatType.OneToOne,
-    chatTypeMsgBelongsTo == ChatType.PrivateGroupChat)
+
+  self.controller.isCommunity(), messageBelongsToActiveSection, chatIdMsgBelongsTo, messageBelongsToActiveChat,
+  message.id, notificationType.int, chatTypeMsgBelongsTo == ChatType.OneToOne,
+  chatTypeMsgBelongsTo == ChatType.PrivateGroupChat)
```

Comment:
Can you undo this change? We use the "tab" to show that those lines belong to the previous call